src/index.ts

```diff
@@ -22,48 +61,17 @@ export = plugin(
       },
       {
         values: {
-          DEFAULT: 'inline-size',
+          DEFAULT: 'size',
```

Comment:
Height-based queries will not work unless using `container-type: size`. I think it would be confusing if height-based queries didn't work out of the box, so I would propose changing the default to `size`.

My understanding is that `inline-size` tells the browser only to consider the inline (width) axis, but `size` tells it to consider both axes. I'm not sure why this distinction is necessary.

I can't think of a good reason not to use `size` as the default - and I don't think this change is breaking, but my knowledge is limited.

If this will be a problem then maybe we need another approach.